cob_hardware_config/cob4-4/config/diagnostics_analyzers.yaml

```diff
@@ -53,7 +53,7 @@ analyzers:
       base:
         type: diagnostic_aggregator/GenericAnalyzer
         path: Base
-        contains: 'base'
+        startswith: '/base'
```

Comment:
For the new bases it should be just `/`, the old base_drive_chain sends `//` for some reason.